LayoutTests/media/audioSession/audioSessionType.html

```diff
@@ -10,6 +10,20 @@
     <body>
         <audio id="audio"></audio>
         <script>
+async function ensureCategory(expected, counter)
+{
+    if (internals.audioSessionCategory() == expected)
+        return "";
+
+    if (!counter)
+        counter = 0;
+    else if (counter > 100)
+        return "ensureCategory '" + title +  "' failed, expected was " + expected + " current value is " + internals.audioSessionCategory();
```

Comment:
Minor nit, a template literal would make this slightly easier to read

Reply:
Done